packages/web-domains/src/common/index.ts

```diff
@@ -4,3 +4,4 @@ export { QueryClientProvider } from './contexts/QueryClientProvider';
 export { RootLayout } from './layout/RootLayout';
 export { GlobalStyle } from './styles/GlobalStyles';
 export { getWebDomain } from './utils/getWebDomain';
+export { Error404 } from './components/Error';
```

Comment:
이거 어차피 Error 경로에 있는 애들 모두 내보내기는 해야할 거 같아서

```suggestion
export * from './components/Error';
```
로 해서 Error 파일 하위에 있는 `index.ts`에서 내보낸 애들 한꺼번에 내보내기 하면 될 거 같아요~!

Reply:
오 알겠습니다~~

Reply:
[6f715dc](https://github.com/depromeet/15th-team3-FE/pull/137/commits/6f715dce67b45639a3669fe08a0e0f920c380104)